### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by vagimeli**
Update list of current maintainers and emeritus maintainers

### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Version
_List the OpenSearch version to which this PR applies, e.g. 2.14, 2.12--2.14, or all._

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kolchfa-aws — CHANGES_REQUESTED** [review]
Please update Stephen Crawford's alias so the maintainers and codeowners list the same one.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -15,3 +14,9 @@ This document lists the maintainers in this repo. See [opensearch-project/.githu
 | David Venable    | [dlvenable](https://github.com/dlvenable)       | Amazon      | 
 | Stephen Crawford | [scraw99](https://github.com/scrawfor99)        | Amazon      |
```
Comment (kolchfa-aws) [inline MAINTAINERS.md:15]:
```suggestion
| Stephen Crawford | [scraw99](https://github.com/stephen-crawford)        | Amazon      |
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -15,3 +14,9 @@ This document lists the maintainers in this repo. See [opensearch-project/.githu
...
 | Eric Pugh        | [epugh](https://github.com/epugh)               | OpenSource Connections  | 
+
+# Emeritus
```
Comment (kolchfa-aws) [inline MAINTAINERS.md:18]:
```suggestion
## Emeritus
```

**Comment by vagimeli** [comment]
@kolchfa-aws Thanks for catching Stephen's username :)

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Pull request merged**